docs/lang/articles/debug/debugging.md

```diff
@@ -190,23 +198,18 @@ x = ti.field(ti.f32, 128)
 def do_sqrt_all():
     for i in x:
         assert x[i] >= 0
-        x[i] = ti.sqrt(x)
+        x[i] = ti.sqrt(x[i])
 ```
 
-When you are done with debugging, simply set `debug=False`. Now `assert`
-will be ignored and there will be no runtime overhead.
+When you are done with debugging, set `debug=False`, and then the program ignores the subsequent `assert` statements and avoid additional runtime overhead.
 
 ## Compile-time `ti.static_assert`
 
 ```python
 ti.static_assert(cond, msg=None)
 ```
 
-Like `ti.static_print`, Taichi also provides a static version of `assert`:
-`ti.static_assert`. It can be useful to make assertions on data types,
-dimensionality, and shapes. It works whether `debug=True` is specified
-or not. When an assertion fails, it will raise an `AssertionError`, just
-like a Python-scope `assert`.
+Like `ti.static_print`, Taichi also provides a static version of `assert`: `ti.static_assert`, which comes handy to assert data types, dimensionality, and shapes. It works regardless of whether `debug=True` is enabled or not. A false statement triggers an `AssertionError`, just as `assert` in the Python scope does.
```

Comment:
With "like `ti.static_print`", I would expect the subject of the main clause to be `ti.static_assert`.

Reply:
“Comes handy" is not the word choice for a technical document.